### ISSUE TITLE
ie not working

### ISSUE THREAD

**Issue by mbradds** [mbradds/pipeline-profiles#77]
Something went wrong with leaflet on commit d925a40f

**Comment by mbradds** [comment]
This can be fixed by using the leaflet cdn for leaflet.css. I would like to find a better solution so that all files originate on CER servers.

**Comment by mbradds** [comment]
This was mostly fixed in 415a32d

Setting URL to false in `webpack.common.js` did the trick

```javascript
      {
        test: /\.css$/,
        include: /node_modules/,
        use: [
          { loader: "style-loader" },
          { loader: "css-loader", options: { url: false } }, // URL false here
        ],
      },
```

Now location services doesn't appear to work in IE 11, but this is a separate issue.
